react/components/ShippingSimulator/index.js

```diff
@@ -77,7 +83,7 @@ const ShippingSimulator = ({
 
   return (
     <Fragment>
-      <div className={`${styles.shippingContainer} t-small c-on-base`}>
+      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress}>
         <AddressRules country={country} shouldUseIOFetching>
           <AddressContainer
             Input={StyleguideInput}
```

Comment:
I checked the Address Form app and the [PostalCodeGetter have a `onSubmit` prop.](https://github.com/vtex/address-form/blob/master/demo/src/App.js#L147) 

We may use it like:

Add the prop `onSubmit={handleClick}` on `PostalCodeGetter`